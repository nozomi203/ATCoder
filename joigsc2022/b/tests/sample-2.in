10
5 J
7 O
10 J
11 G
12 J
13 I
17 J
18 J
19 J
20 J
4
4 9
15 14
6 20
7 20
