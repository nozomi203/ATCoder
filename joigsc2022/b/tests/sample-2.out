13
19
20
21
